life_server/functions.sqf

```diff
@@ -412,4 +412,24 @@ TON_fnc_MapMarkersAdmin = compileFinal "
   } forEach playableUnits;
 ";
 
-publicVariable "TON_fnc_MapMarkersAdmin";
\ No newline at end of file
+publicVariable "TON_fnc_MapMarkersAdmin";
+
+life_fnc_vehicleKind =
+compileFinal "
+    private [""_par"",""_veh""];
+    _par = _this select 0;
+    if (typeName _par == ""STRING"") then {
+    _veh = _par createVehicle [0,0,0];
+    _veh enableSimulation false;
+    } else { if (typeName _par == ""OBJECT"") then { _veh = _par; }; };
+    if (isNull _veh) exitWith {};
+   _kind= switch (true) do {
+        case (_veh isKindOf ""LandVehicle""): {""Car""};
+        case (_veh isKindOf ""Air""): {""Air""};
+        case (_veh isKindOf ""Ship""): {""Ship""};
+    };
+    if (typeName _par == ""STRING"") then { deleteVehicle _veh; };
+    _kind;
+
+";
++publicVariable "life_fnc_vehicleKind";
```

Comment:
There is a `+` at the [start of this line](https://github.com/DardoTheMaster/Framework/blob/b6ff7ac343c61a00c981c8df7b07ad5964120718/life_server/functions.sqf#L435).